packages/clerk-js/src/ui/signUp/SignUpStart.tsx

```diff
@@ -165,7 +165,7 @@ function _SignUpStart(): JSX.Element {
     e.preventDefault();
 
     const fieldsToSubmit = Object.entries(fields).reduce(
-      (acc, [k, v]) => [...acc, ...(v && formState[k as FormStateKey]?.value ? [formState[k as FormStateKey]] : [])],
+      (acc, [k, v]) => [...acc, ...(v && formState[k as FormStateKey] ? [formState[k as FormStateKey]] : [])],
```

Comment:
🙃 Feel free to disregard my opinion since my exposure to TS/JS is very limited: I find this code very hard to read. Perhaps it could be refactored somehow?